src/Tools/dotnet-dump/dotnet-dump.csproj

```diff
@@ -20,6 +20,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Diagnostics.Runtime" Version="$(MicrosoftDiagnosticsRuntimeVersion)" />
     <PackageReference Include="Microsoft.SymbolStore" Version="$(MicrosoftSymbolStoreVersion)" />
+    <PackageReference Include="ParallelStacks.Runtime" Version="2.0.1" />
```

Comment:
We usually add a msbuild build property (i.e. ParalletStacksRuntimeVersion) to eng/Versions.props for external packages.

Reply:
Good point: using ParallelStacksRuntimeVersion in the last commit.